dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -517,7 +529,22 @@
          * Show link to Terms of Use page in the footer (if set to 'true').
          * By default is set to 'false', so link won't be shown.
          */
-        ShowTermsOfUseFooterLink
+        ShowTermsOfUseFooterLink,
+        
+        /**
+         * Name of the site that will be presented in the header.
+         * Setting can be postfixed with language code to
+         * obtain translated versions.
+         */
+        SiteName,
+        
+        /**
+         * Fullname of the site that will be presented in the header
+         * below {@link Key#SiteName}.
+         * Setting can be postfixed with language code to
+         * obtain translated versions.
+         */
+        SiteFullname
```

Comment:
`SiteFullName`?

Reply:
Well. I was wondering about naming. The way we show SiteName is by shortcut (for example. RepOD). And Fullname is expanded, without shortcut (Repository for Open Data)

Reply:
I'm not 100% sure that `SiteFullName` is better. By naming conventions alone maybe yes, but looking at usage maybe not. 
I'll leave it to you if you want to change that.